contracts/src/factory/facets/architect/ArchitectBase.sol

```diff
@@ -53,8 +54,29 @@ abstract contract ArchitectBase is Factory, IArchitectBase, PricingModulesBase {
     return ArchitectStorage.layout().spaceByTokenId[tokenId];
   }
 
+  function _createSpaceWithPrepay(
+    CreateSpace memory createSpace
```

Comment:
```suggestion
    SpaceInfo calldata spaceInfo,
    Prepay calldata prepay
```
I'd say we change `SpaceInfo` to
```solidity
  struct SpaceInfo {
    Metadata metadata;
    Membership membership;
    ChannelInfo channel;
  }
```
for consistency and pass `Prepay` separately so as to keep everything in calldata.

Reply:
we're moving from using SpaceInfo to CreateSpace, while keeping backwards compatibility

Reply:
client side already uses these types for old create space so we can't update the current struct without breaking